app-misc/neofetch/neofetch-5.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="A fast, highly customizable system info script"
+DESCRIPTION="Neofetch displays information about your system next to an image, \
+your OS logo, or any ASCII file of your choice. The main purpose of Neofetch \
+is to be used in screenshots to show other users what OS/Distro you're \
+running, what Theme/Icons you're using etc"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+COPYRIGHT="2016-2018 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/dylanaraps/$portName/archive/$portVersion/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="2a4f4853bf83b88a037994dbc53a90c8bd5708f5eeb3392f56d4e49c49d995b3"
+
+ARCHITECTURES="any"
+
+USER_SETTINGS_FILES="$relativeSettingsDir/neofetch/config.conf"
+
+PROVIDES="
+	neofetch
+	cmd:neofetch
+	"
+REQUIRES="
+	haiku
+	cmd:bash
+	cmd:sed
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i \
+		-e "s|/share/neofetch/|/$relativeDataDir/neofetch/|;" \
+		Makefile
+}
+
+INSTALL()
+{
+	make install PREFIX=$prefix MANDIR=$manDir SYSCONFDIR=$sysconfDir
```

Comment:
```diff
-	make install PREFIX=$prefix MANDIR=$manDir SYSCONFDIR=$sysconfDir
+	make install PREFIX=$prefix MANDIR=$manDir
```

`SYSCONFDIR` is not used by the [`Makefile`](https://github.com/dylanaraps/neofetch/blob/5.0.0/Makefile)